img/v/vaultcord.com.svg

```diff
@@ -0,0 +1 @@
 155.033 265.051,154.927 265.600,155.031 265.600 C 155.301 265.600,156.056 263.522,155.857 263.324 C 155.767 263.233,155.501 263.708,155.267 264.380 M153.265 269.017 C 152.595 270.488,152.723 270.802,153.447 269.465 C 153.786 268.840,153.988 268.254,153.896 268.163 C 153.805 268.071,153.521 268.456,153.265 269.017 M326.600 269.414 C 325.595 269.898,325.563 269.953,326.400 269.751 C 326.950 269.619,327.571 269.350,327.780 269.155 C 328.298 268.671,328.014 268.733,326.600 269.414 M126.200 271.200 L 99.800 271.409 126.076 271.504 C 143.266 271.567,152.438 271.462,152.600 271.200 C 152.736 270.980,152.792 270.843,152.724 270.896 C 152.656 270.948,140.720 271.085,126.200 271.200 M316.800 272.400 C 315.996 272.657,315.957 272.723,316.600 272.732 C 317.040 272.738,317.760 272.589,318.200 272.400 C 319.219 271.962,318.168 271.962,316.800 272.400 M313.000 273.200 C 311.893 273.482,311.877 273.509,312.800 273.547 C 313.350 273.569,314.070 273.413,314.400 273.200 C 315.081 272.760,314.729 272.760,313.000 273.200 M268.200 274.381 C 268.968 274.729,271.110 274.956,270.767 274.653 C 270.675 274.572,269.880 274.396,269.000 274.262 C 267.885 274.093,267.642 274.129,268.200 274.381 M306.000 274.400 C 304.656 274.831,306.108 274.831,307.800 274.400 C 308.878 274.126,308.898 274.094,308.000 274.087 C 307.450 274.083,306.550 274.224,306.000 274.400 M276.900 275.491 C 277.725 275.578,279.075 275.578,279.900 275.491 C 280.725 275.405,280.050 275.334,278.400 275.334 C 276.750 275.334,276.075 275.405,276.900 275.491 M294.500 275.494 C 295.545 275.576,297.255 275.576,298.300 275.494 C 299.345 275.412,298.490 275.344,296.400 275.344 C 294.310 275.344,293.455 275.412,294.500 275.494 M286.707 275.890 C 287.426 275.979,288.506 275.977,289.107 275.886 C 289.708 275.795,289.120 275.722,287.800 275.724 C 286.480 275.726,285.988 275.801,286.707 275.890 " stroke="none" fill="#959df6" fill-rule="evenodd"></path><path id="path4" d="M295.700 177.076 C 296.085 177.176,296.715 177.176,297.100 177.076 C 297.485 176.975,297.170 176.893,296.400 176.893 C 295.630 176.893,295.315 176.975,295.700 177.076 M303.300 177.076 C 303.685 177.176,304.315 177.176,304.700 177.076 C 305.085 176.975,304.770 176.893,304.000 176.893 C 303.230 176.893,302.915 176.975,303.300 177.076 M204.109 208.000 C 204.109 208.990,204.187 209.395,204.282 208.900 C 204.378 208.405,204.378 207.595,204.282 207.100 C 204.187 206.605,204.109 207.010,204.109 208.000 M299.300 237.076 C 299.685 237.176,300.315 237.176,300.700 237.076 C 301.085 236.975,300.770 236.893,300.000 236.893 C 299.230 236.893,298.915 236.975,299.300 237.076 M282.307 275.490 C 283.026 275.579,284.106 275.577,284.707 275.486 C 285.308 275.395,284.720 275.322,283.400 275.324 C 282.080 275.326,281.588 275.401,282.307 275.490 M290.705 275.483 C 291.202 275.579,291.922 275.575,292.305 275.475 C 292.687 275.375,292.280 275.296,291.400 275.301 C 290.520 275.305,290.207 275.387,290.705 275.483 " stroke="none" fill="#acb4fc" fill-rule="evenodd"></path></g></svg>
```

Comment:
It looks like you've run a PNG through some sort of vectorization tool. Is that correct?
Vector images can be turned into rasterized images, but the opposite can't be done in a good way.
If you have the original vector graphics somewhere, try converting it to SVG. If not, replace this image with a PNG image that's 32x32, 64x64 or 128x128 pixels and compress it with [TinyPNG](https://tinypng.com/) before submitting.